content/learning-paths/servers-and-cloud-computing/cca-veraison/attestation-token.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Download and inspect the attestation token
+weight: 4
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+In this section, you will obtain an example CCA attestation token.
+
+## Install Go
+
+In order to run the tools used for attestation, start by installing the Go language on your system. First, you will remove any existing Go installation. After that, you obtain the install files and
+
+```bash
+rm -rf /usr/local/go
+
+wget https://go.dev/dl/go1.23.3.linux-$(dpkg --print-architecture).tar.gz
+tar -C /usr/local -xzf go1.23.3.linux-$(dpkg --print-architecture).tar.gz
+```
+
+Export the installation path and add it to your $PATH environment variable.
+
+```bash
+export PATH=$PATH:/usr/local/go/bin
+```
+Verify the installation by checking that the command outputs the installed version.
+
+```bash
+go version
+```
+
+## Install Git
+
+Verify that `git` is installed using the command below. It should output the version available on your computer.
+
+```bash
+git --version
+```
+
+## Download the Example CCA Attestation Token
+
+Using your preferred web browser, navigate to the [token in the TrustedFirmware-M tools repository](https://github.com/TrustedFirmware-M/tf-m-tools/blob/main/iat-verifier/tests/data/cca_example_token.cbor)
+
+Use GitHub’s download button, located on the right of the upper toolbar, to download the token as a *raw* (binary) file.
+
+![download_raw.png](./download_raw.png)
+
+Place this file in the `$HOME` folder, while keeping the file name the same. The rest of this learning path will use the notation `$HOME/cca_example_token.cbor` as the file path.
+
+{{% notice Note %}}
+You will notice that the filename extension on the example token is `.cbor`, which also denotes the format of the data. CBOR is the Concise Binary Object Representation. You are likely to already be familiar with JSON (the JavaScript Object Notation). JSON provides a standard way to convey nested structures of key-value pairs. CBOR is conceptually the same as JSON. The difference is that CBOR is a binary format, rather than a text-based format like JSON. CBOR is designed for compactness and ease of machine-readability, but at the expense of human-readability. You can learn more about CBOR here.
```

Comment:
There's a link missing: "You can learn more about CBOR here"

The "here" should link to [here](https://cbor.io)

Reply:
done